runtime/kreivo/src/payments.rs

```diff
@@ -28,29 +29,39 @@ impl pallet_payments::BenchmarkHelper<AccountId, FungibleAssetLocation, Balance>
 
 pub struct KreivoFeeHandler;
 
-const MANDATORY_FEE: bool = true;
 pub const SENDER_FEE: Percent = Percent::from_percent(1);
 pub const BENEFICIARY_FEE: Percent = Percent::from_percent(3);
 pub const INCENTIVE_PERCENTAGE: u8 = 10;
 
 impl FeeHandler<Runtime> for KreivoFeeHandler {
 	fn apply_fees(
 		asset: &AssetIdOf<Runtime>,
-		_sender: &AccountId,
-		_beneficiary: &AccountId,
+		sender: &AccountId,
+		beneficiary: &AccountId,
 		amount: &Balance,
 		_remark: Option<&[u8]>,
 	) -> Fees<Runtime> {
 		let min = <Assets as fungibles::Inspect<AccountId>>::minimum_balance(*asset);
+
+		let is_sender_paying_fees:bool = match PalletId::try_from_sub_account::<CommunityId>(sender) {
+			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => true,
+			_ => false
+		};
+
+		let is_beneficiary_paying_fees:bool = match PalletId::try_from_sub_account::<CommunityId>(beneficiary) {
+			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => true,
+			_ => false
+		};
```

Comment:
Same as reviewed in #379: Community Accounts don't pay fees, others do.

```suggestion
		let is_sender_paying_fees:bool = match PalletId::try_from_sub_account::<CommunityId>(sender) {
			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => false,
			_ => true
		};

		let is_beneficiary_paying_fees:bool = match PalletId::try_from_sub_account::<CommunityId>(beneficiary) {
			Some((pallet_id, _)) if pallet_id == crate::communities::CommunityPalletId::get() => false,
			_ => true
		};
```